app/services/prometheus/service_test.go

```diff
@@ -64,7 +67,7 @@ func Test_ConstructMetricName_Native(t *testing.T) {
 
 	mocks.MAssetsService.On("IsNative", sourceNetworkId, assetAddress).Return(true)
 
-	expectedNative := fmt.Sprintf("%v_%v_to_%v_%v_%v", constants.Native, "Hedera", "Network3", "0_0_1234_1234_1234", constants.MajorityReachedNameSuffix)
```

Comment:
Maybe we can set the `"0_0_1234_1234_1234"` also as a constant, since it is used in both `Test_ConstructMetricName_Native` and `Test_ConstructMetricName_Wrapped`?